src/stories/Forms/Input/index.tsx

```diff
@@ -39,69 +37,46 @@ export interface InputProps {
     | 'time'
     | 'url'
     | 'week';
-  label?: string;
-  value?: string;
-  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void; // eslint-disable-line no-unused-vars
-  isRequired?: boolean;
-  alertLabel?: string;
-  helpLink?: string;
-  helpLabel?: string;
-  className?: string;
 }
 
 /**
  * Input component
  */
 export const Input = React.forwardRef(
-  (
-    {
+  (props: InputProps, ref: React.RefObject<HTMLInputElement>) => {
+    const {
+      ariaDescribedBy,
+      ariaErrorMessage,
+      className,
+      hasError,
       id,
+      isRequired,
+      name,
       type = 'text',
-      label,
       value,
-      onChange,
-      isRequired,
-      alertLabel,
-      helpLink,
-      helpLabel,
-      className,
-    }: InputProps,
-    ref: React.RefObject<HTMLInputElement>
-  ) => {
+      ...additionalInputProps
+    } = props;
+
     return (
       <Wrapper
-        id={id}
+        aria-hidden={type === 'hidden'}
```

Comment:
You need to learn my about these aria things. Very nice that you have added it 👍